network/src/counters.rs

```diff
@@ -1,6 +1,43 @@
 // Copyright (c) The Libra Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
+//! # Metrics
```

Comment:
This is great but seems like something we should either have in our metrics crate or just point to the original source documentation. Being in network means other counter crates won't benefit.

Reply:
I agree.  I haven't spent a lot of time looking into common metrics infrastructure, in comparison to how we do logging, so I want a place to put this, just don't know where yet

Reply:
Found it!  will update PR.